extensions/transfer/transfer-nifi/src/main/java/com/microsoft/dagx/transfer/nifi/NifiTransferExtension.java

```diff
@@ -18,9 +18,11 @@
 
 public class NifiTransferExtension implements ServiceExtension {
     @DagxSetting
-    private static final String URL_SETTING = "nifi.url";
+    private static final String URL_SETTING = "dagx.nifi.url";
+    private static final String URL_SETTING_FLOW = "dagx.nifi.flow.url";
 
     private static final String DEFAULT_NIFI_URL = "http://localhost:8080";
+    private static final String DEFAULT_NIFI_FLOW_URL = "http://localhost:8888";
```

Comment:
Maybe I don't know NiFi good enough. Is http://localhost:8888 a default endpoint or does this only work on already pre-configured NiFi instanced?

Reply:
that has to do with how the Nifi flow is configured: it has an `HttpHandleRequest` processor as ingress, which listens on `:8888/contentListener`